app/components/Comments/Source.jsx

```diff
@@ -30,13 +30,15 @@ const PLAYER_CONFIG = {youtube: {playerVars: { showinfo: 1 }}}
 
 export const Source = ({ source: { url, title, site_name }, withoutPlayer }) => {
   if (!withoutPlayer && isPlayer(url)) {
+    const maxVolume = 1
```

Comment:
pls do not use a variable just for that...
If it's for using it everywhere you could set it globally as a constant but I wouldn't recommend so here